### PR TITLE
Bugfix/df 1814 bfs code already referenced bug

### PR DESCRIPTION
**Pull request by ganqqwerty**
image

### CODE REVIEW

**Comment by cjpsycos** [comment]
you should add the id of the chFiche

**Comment by ganqqwerty** [comment]
> you should add the id of the chFiche

Done

File: alv-portal-ui/src/app/competence-catalog/ch-fiches/ch-fiches.module.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ import { ChFicheTitleModalComponent } from './ch-fiche-title-modal/ch-fiche-titl
     SharedModule,
     ChFichesRoutingModule,
     InfiniteScrollModule,
-    SharedCompetenceCatalogModule
+    SharedCompetenceCatalogModule,
```
Comment (cjpsycos) [inline alv-portal-ui/src/app/competence-catalog/ch-fiches/ch-fiches.module.ts]: Why do you add always a coma to the last element of lists?

File: alv-portal-ui/src/app/competence-catalog/shared/interactive-list-item/interactive-list-item.component.ts [hunk 2]
```diff
@@ -37,7 +37,7 @@ export class InteractiveListItemComponent implements OnInit {
   constructor(private i18nService: I18nService) {
   }
 
-  private _multiLanguageTitle: TranslatedString;
+  private _multiLanguageTitle: TranslatedString; // todo use multi-language-string component
```
Comment (cjpsycos) [inline alv-portal-ui/src/app/competence-catalog/shared/interactive-list-item/interactive-list-item.component.ts]: what means that comment?

File: alv-portal-ui/src/app/competence-catalog/shared/multi-language-string/multi-language-string.component.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+import { I18nService } from '../../../core/i18n.service';
+
+@Component({
+  selector: 'alv-multi-language-string',
+  templateUrl: './multi-language-string.component.html',
+  styleUrls: ['./multi-language-string.component.scss']
+})
+export class MultiLanguageStringComponent {
+
+  titleViewValue$: Observable<TranslatedStringToCurrentLanguage>;
```
Comment (cjpsycos) [inline alv-portal-ui/src/app/competence-catalog/shared/multi-language-string/multi-language-string.component.ts]: if it is a shared class, it should be neutral.
it's not only used for title, but also for description.
use name like e.g. viewValue and valueModel

File: alv-portal-ui/src/app/competence-catalog/shared/interactive-list-item/interactive-list-item.component.ts (hunk 2 above)
Comment (ganqqwerty) [reply alv-portal-ui/src/app/competence-catalog/shared/interactive-list-item/interactive-list-item.component.ts]: I created a separate component for multi-language string. Here it's not used, but it should be used in the future

File: alv-portal-ui/src/app/competence-catalog/shared/multi-language-string/multi-language-string.component.ts (hunk 3 above)
Comment (ganqqwerty) [reply alv-portal-ui/src/app/competence-catalog/shared/multi-language-string/multi-language-string.component.ts]: ok

File: alv-portal-ui/src/app/competence-catalog/ch-fiches/ch-fiches.module.ts (hunk 1 above)
Comment (ganqqwerty) [reply alv-portal-ui/src/app/competence-catalog/ch-fiches/ch-fiches.module.ts]: I will check my IDE settings, thanks

**Pull request merged**